src/main/java/org/jitsi/videobridge/TransformChainManipulator.java

```diff
@@ -0,0 +1,27 @@
+package org.jitsi.videobridge;
```

Comment:
Could you please add the copyright notice here?

Reply:
I added the copyright notice.